python/pyarrow/tests/test_csv.py

```diff
@@ -220,6 +221,18 @@ def test_simple_nulls(self):
             'e': [b"3", b"nan", b"\xff"],
             }
 
+    def test_simple_timestamps(self):
+        # Infer a timestamp column
+        rows = b"a,b\n1970,1970-01-01\n1989,1989-07-14\n"
```

Comment:
Shouldn't this be a `date` column and only a `datetime` column when it includes hours/minutes?

Reply:
What about adding a time to these so that we don't have a test that would break when we add `date` support?

Reply:
Well, we can fix the test by then. Right now those are inferred as timestamps, and that's what the test checks for.